src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import axios from 'axios';
+import './Subreddit-module.css';
+import Header from '../../componenten/header/Header';
+import { ReactComponent as BackIcon } from '../../assets/back.svg';
+
+function Subreddit() {
+    const [details, setDetails] = useState({});
+    const { id } = useParams();
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const response = await axios.get(`https://www.reddit.com/r/${id}/about.json`);
+                console.log(response);
+                setDetails(response.data.data);
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchData();
+    }, []);
+
+    return (
+        <>
+            <Header>
+                <h1>r/{id}</h1>
+                <h4>Subreddit specifications</h4>
+            </Header>
+            <main>
+                <section className="outer-content-container subreddit-specifications">
+                    <div className="inner-content-container">
+                        {Object.keys(details).length > 0 && (
```

Comment:
Je check kan dan hier simpeler:
```suggestion
                        {details && (
```